### PR TITLE
Fixes from game jam

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

File: _includes/home.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Read [Installation guide]({{ site.baseurl }}{% post_url /getting-started/2018-01
 
 ## Create your first scene
```
Comment (eordano) [inline _includes/home.md:51]: Following command line command?

File: _posts/development-guide/2018-02-13-3d-model-animations.md [hunk 2]
```diff
@@ -179,10 +179,10 @@ To stop an animation and set the posture back to the first frame in the animatio
 clipSwim.stop()
 ```
 
-To play an animation from the start, regardless of what frame the animation is currently in, use the `restart()` function of the `AnimationState` object.
+To play an animation from the start, regardless of what frame the animation is currently in, use the `reset()` function of the `AnimationState` object.
 
 ```ts
-clipSwim.restart()
+clipSwim.reset()
```
Comment (eordano) [inline _posts/development-guide/2018-02-13-3d-model-animations.md:185]: Restart is no longer a thing, right?

File: _posts/development-guide/2018-02-14-click-events.md [hunk 3]
```diff
@@ -52,6 +52,32 @@ myEntity.addComponent(
...
+
+You can detect which button was clicked, by checking the `buttonId` field from the event data. The value of this field will either return a _0_, _1_ or _2_, which map to the `POINTER`, `PRIMARY`, and `SECONDARY` buttons respectively.
+
+```ts
+const myEntity = new Entity()
+myEntity.addComponent(new BoxShape())
+
+myEntity.addComponent(
+  new OnPointerDown(e => {
+	if (e.pointerId == 0){
```
Comment (eordano) [inline _posts/development-guide/2018-02-14-click-events.md:67]: Typo: missing "

**Review by eordano — CHANGES_REQUESTED** [review]

File: _posts/development-guide/2018-02-13-3d-model-animations.md (hunk 2 above)
Comment (nearnshaw) [reply _posts/development-guide/2018-02-13-3d-model-animations.md:185]: Nope, no longer used at all

**Comment by nearnshaw** [comment]
@eordano    all your comments are now addressed

**Review by eordano — APPROVED** [review]

**Pull request merged**